### PR TITLE
Mds 2347 now lists

### PR DESCRIPTION
**Pull request by nayr974**
This PR updates all NoW lists ("Provincial Reporting"  / Mines -> Notices Of Work) to use the Core now_application_identity table, utilizing a view to return values if the notices of work have not been imported into Core.

### CODE REVIEW

File: frontend/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js [hunk 1]
```diff
@@ -138,12 +137,12 @@ export class NoticeOfWorkTable extends Component {
     {
       title: "NoW No.",
       dataIndex: "nowNum",
-      sortField: "trackingnumber",
+      sortField: "tracking_number",
       render: (text, record) => (
         <Link to={router.NOTICE_OF_WORK_INITIAL_APPLICATION.dynamicRoute(record.key)}>{text}</Link>
       ),
```
Comment (mmumby) [inline frontend/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js]: Does this link still work? / does the component for the initial submission render correctly?

File: frontend/src/tests/components/mine/NoticeOfWork/__snapshots__/MineNoticeOfWorkTable.spec.js.snap [hunk 2]
```diff
@@ -67,24 +67,22 @@ exports[`MineNoticeOfWorkTable renders properly 1`] = `
     dataSource={
       Array [
         Object {
-          "date": "Aug 14 2019",
-          "key": "07e801a0-fa33-4c3b-abcc-ac6df628d483",
+          "date": "N/A",
+          "key": undefined,
```
Comment (Jsyro) [inline frontend/src/tests/components/mine/NoticeOfWork/__snapshots__/MineNoticeOfWorkTable.spec.js.snap]: this looks like mocks needs to be updated.

File: frontend/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js (hunk 1 above)
Comment (nayr974) [reply frontend/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js]: yes, the link continues to work for the initial submission 👍

File: frontend/src/tests/components/mine/NoticeOfWork/__snapshots__/MineNoticeOfWorkTable.spec.js.snap (hunk 2 above)
Comment (nayr974) [reply frontend/src/tests/components/mine/NoticeOfWork/__snapshots__/MineNoticeOfWorkTable.spec.js.snap]: Good catch; will update

Comment (nayr974) [reply frontend/src/tests/components/mine/NoticeOfWork/__snapshots__/MineNoticeOfWorkTable.spec.js.snap]: Updated

**Pull request merged**